src/components/calendar-components/AddEvent.js

```diff
@@ -1,56 +1,95 @@
-import React from 'react';
+import React, {Component} from 'react';
 import PropTypes from 'prop-types';
 import {ParticipantsSelect} from './ParticipantsSelect';
 import {TypeSelect} from './TypeSelect';
 import TextField from '@material-ui/core/TextField';
 import Button from '@material-ui/core/Button';
-import {getDateFormatYMD} from './../../helpers/calendar-helper';
+import {getDateFormatYMD, getTimeFormat, isValidParticipants, isValidType} from "../../helpers/calendar-helper";
 
 
-export const AddEvent = (props) => {
-  const date = props.date;
-  const handleSubmit= () => {
-    props.onSaveButtonClick(document.getElementById("date").value,
-      document.getElementById("start").value,
-      document.getElementById("end").value,
-      document.getElementById("participants").value,
-      document.getElementById("type").value)
+export class AddEvent extends Component {
+
+  constructor(props) {
+    super(props);
+    let defaultStart = "09:00";
+    let defaultEnd = "10:00";
+    if (props.eventToEdit) {
+      defaultStart = getTimeFormat(props.eventToEdit.start);
+      defaultEnd = getTimeFormat(props.eventToEdit.end);
+    }
+    this.state = {
+      date: getDateFormatYMD(props.date),
+      start: defaultStart,
+      end: defaultEnd,
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const participants = this.state.participants;
+    const type = this.state.type;
+    const participantsSelected = isValidParticipants(participants);
+    const typeSelected = isValidType(type);
+    if (!participantsSelected) {
+        console.log("not valid participants");
```

Comment:
We could figure out how to use form validation for this, but this should work for now.